samples/smpe_list.py

```diff
@@ -0,0 +1,181 @@
+#! /usr/bin/env python3
+'''
+List SMPE Data. This is a Python implementation that provides the ability to
+list data stored in SMPE. It relies on yaml to get defaults and argparse to
+handle input.
+It would replace the following JCL:
+//SMPLIST  JOB ,,MSGLEVEL=1,MSGCLASS=H,CLASS=A,REGION=0M,
+//             NOTIFY=&SYSUID
+//******************************************************************
+//SMPLST   EXEC PGM=GIMSMP
+//SMPCSI   DD  DSN=AS4SMP.GLOBAL.CSI,DISP=SHR
+//SMPLOG   DD  DUMMY
+//SMPLOGA  DD  DUMMY
+//SMPWRK6  DD DSN=&&TEMP,
+//       SPACE=(CYL,(2500,100,500)),VOL=SER=USRAT5,UNIT=3390,
+//       DCB=(DSORG=PO,RECFM=FB,LRECL=80,BLKSIZE=3200),
+//       DISP=(NEW,DELETE)
+//SMPCNTL  DD  *
+ SET BDY(GLOBAL).
+ LIST.
+ '''
+import sys
+import os
+import yaml
+import argparse
+from zoautil_py import mvscmd, datasets
+from zoautil_py.types import DDStatement, DatasetDefinition, FileDefinition
+
+
+def SMPE_list(target_zone="GLOBAL", list_options=None,
+              high_level_qualifier="SYS1"):
+    '''
+    This function does the heavy lifting. It performs the function that
+        is contained in the JCL. It sets up the DD statements and issues the
+        call to the executable.
+    '''
+
+    # Initialize DD List
+    dd_list = []
+
+    # and the temporary dataset
+    temp_dataset = None
+
+    # get the defaults from the yaml file
+    defaults = getdefaults("./SMPElistDefaults.yaml")
```

Comment:
Follow on to the comment on line 109 about changing `getdefaults` to `get_defaults`